docs/tutorials/extending_admin_ui/2_creating_a_dashboard_tab.md

```diff
@@ -0,0 +1,133 @@
+
+use eZ\Publish\API\Repository\SearchService;
+use eZ\Publish\Core\Pagination\Pagerfanta\ContentSearchAdapter;
+use EzSystems\EzPlatformAdminUi\Tab\AbstractTab;
+use EzSystems\EzPlatformAdminUi\Tab\OrderedTabInterface;
+use EzSystems\EzPlatformAdminUi\Tab\Dashboard\PagerContentToDataMapper;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use Pagerfanta\Pagerfanta;
+use Symfony\Component\Translation\TranslatorInterface;
+use Twig\Environment;
+
+class EveryoneArticleTab extends AbstractTab implements OrderedTabInterface
+{
+    /** @var PagerContentToDataMapper */
+    protected $pagerContentToDataMapper;
+
+    /** @var SearchService */
+    protected $searchService;
+
+    public function __construct(
+        Environment $twig,
+        TranslatorInterface $translator,
+        PagerContentToDataMapper $pagerContentToDataMapper,
+        SearchService $searchService
+    ) {
+        parent::__construct($twig, $translator);
+
+        $this->pagerContentToDataMapper = $pagerContentToDataMapper;
+        $this->searchService = $searchService;
+    }
+
+    public function getIdentifier(): string
+    {
+        return 'everyone-article';
+    }
+
+    public function getName(): string
+    {
+        return 'Articles';
+    }
+
+    public function getOrder(): int
+    {
+        return 300;
+    }
+
+    public function renderView(array $parameters): string
+    {
+        $page = 1;
+        $limit = 10;
+
+        $query = new LocationQuery();
+
+        $query->sortClauses = [new SortClause\DateModified(LocationQuery::SORT_DESC)];
+        $query->query = new Criterion\LogicalAnd([
+            new Criterion\ContentTypeIdentifier('article'),
+        ]);
+
+        $pager = new Pagerfanta(
+            new ContentSearchAdapter($query,
+                $this->searchService
+            )
+        );
+        $pager->setMaxPerPage($limit);
+        $pager->setCurrentPage($page);
+
+        return $this->twig->render('EzPlatformAdminUiBundle:dashboard/tab:all_content.html.twig', [
+            'data' => $this->pagerContentToDataMapper->map($pager),
+        ]);
+    }
+}
+```
+
+!!! tip
+
+    The tab extends `AbstractTab`.
+    There are also [other tab types that you can extend](../../guide/extending_ez_platform_ui.md#adding-a-new-tab).
+
+It also implements `OrderedTabInterface` (see line 15), which enables you to define the order in which the tab is displayed in the Dashboard.
+This is done using the `getOrder` method (see line 45).
+
+The rendering is done using the built-in `all_content.html.twig` template,
+which ensures the tab looks the same as the existing tabs.
+
+## Check results
+
+At this point you can go to the Dashboard in the Back Office.
+In the "Everyone" block you can see the new "Articles" tab with the first ten articles in the Repository.
```

Comment:
maybe it would be good to add a note that 'if you do not see the changes you should refresh the page and/or clear the cache'